metadata-ingestion/src/datahub/utilities/advanced_thread_executor.py

```diff
@@ -32,7 +32,7 @@ def __init__(self, max_workers: int, max_pending: int) -> None:
 
         It works similarly to a ThreadPoolExecutor, with the following changes:
         - At most one request per partition key will be executing at a time.
-        - If the number of pending requests exceeds the threshold, the submit call
+        - If the number of pending requests exceeds the threshold, the submitted call
```

Comment:
maybe this is more clear
```suggestion
        - If the number of pending requests exceeds the threshold, the submit() call
```